index.html

```diff
@@ -3186,21 +3186,25 @@ <h3>Terms of Use</h3>
         </p>
 
         <p>
-The value of the <code>termsOfUse</code> <a>property</a> tells the
-<a>verifier</a> what actions it is required to perform (an <em>obligation</em>),
+Examples for the value of the <code>termsOfUse</code> <a>property</a> may be
```

Comment:
```suggestion
Examples for the value of the <code>termsOfUse</code> <a>property</a> might be
```

Avoid the use of RFC terms when not making RFC statements.

Reply:
done

Reply:
I added a different second example to replace the obligation based one that you suggested by removed. See if you like it.